Assets/HoloToolkit/Input/Scripts/Cursor/Cursor.cs

```diff
@@ -190,6 +197,8 @@ protected virtual void OnDisable()
             TargetedObject = null;
             TargetedCursorModifier = null;
             visibleHandsCount = 0;
+            IsHandVisible = false;
+            OnCursorStateChange(CursorStateEnum.Contextual);
```

Comment:
Would we just want to change to context to None here as well?
What purpose does contextual have?